src/common/schemas/specifications/check-create.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "checkCreate",
+  "link": "check-create",
+  "type": "object",
+  "properties": {
+    "destination": {
+      "$ref": "address",
+      "description": "Address of the account that can cash the check."
+    },
+    "sendMax": {
+      "$ref": "laxAmount",
+      "description": "Amount of source currency the check is allowed to debit the sender, including transfer fees on non-XRP currencies."
+    },
+    "destinationTag": {
+      "$ref": "tag",
+      "description": "Destination tag that identifies the reason for the check, or a hosted recipient to pay."
+    },
```

Comment:
I'll mention is passing that a "sourceTag" is also an allowed field.  But that's allowed in any transaction, so you may be handling that elsewhere.